src/test/groovy/uk/gov/justice/digital/hmpps/keyworker/integration/specs/RoleAssignmentSpecification.groovy

```diff
@@ -25,7 +26,15 @@ class RoleAssignmentSpecification extends TestSpecification {
 
         then: "The request is accepted"
 
-        response.statusCode == HttpStatus.NO_CONTENT
+        response.statusCode == HttpStatus.OK
+        def roleAssignmentStats = jsonSlurper.parseText(response.body)
+        def moorlandResults = roleAssignmentStats['MDI']
+        moorlandResults != null
+        moorlandResults.numAssignRoleSucceeded == 0
```

Comment:
would be nicer if there was a single comparison of the stats object rather than each of the fields individually

Reply:
now comparing returned map